Makefile.include

```diff
@@ -202,6 +202,14 @@ include $(RIOTMAKE)/boards.inc.mk
 # Debug targets for build system migration
 include $(RIOTMAKE)/dependencies_debug.inc.mk
 
+# List all available applications
+include $(RIOTMAKE)/app_dirs.inc.mk
+
+ifeq (,$(filter $(APPDIR),$(APPLICATION_DIRS_ABSOLUTE)))
```

Comment:
Wouldn't this be screaming in the face of every user of an application built out of tree?